fixtures/u2_import_subsymbols_namespaced.js

```diff
@@ -1,2 +1,3 @@
 import * as namespace from 'file';
 import type File from "file";
+import type Environment from "react-relay";
```

Comment:
Any reason to add this line? Looks structurally the same as the previous one.

Reply:
removed in 7b65c80